riptide-failsafe/README.md

```diff
@@ -113,6 +113,26 @@ Http.builder().requestFactory(new HttpComponentsClientHttpRequestFactory())
     .build();
 ```
 
+### Custom executor
```

Comment:
When looking at the preview of the README file, I noticed that we still link to the `failsafe-actuator` project. Given that it was archived in December of last year, I don't know if we should keep this or not. @fatroom Any opinion?

Reply:
This is a bit broad question, I'm considering either merge that project, or take over maintenance of it.